### PR TITLE
New Simple Bank exercise

### PR DESCRIPTION
**Pull request by doomspork**

### CODE REVIEW

**Comment by doomspork** [comment]
Part 2 (to add in new topics):  Online Banking.  Extend this exercise to include a REST API.

File: simple_bank/test/simple_bank_test.exs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      {:error, :existing_account} = SimpleBank.register(bank_pid, "Test")
+    end
+  end
+
+  describe "deposit/3" do
+    test "increases the account balance by the deposited amount", %{bank: bank_pid}  do
+      assert {:ok, 110} == SimpleBank.deposit(bank_pid, "test_id", 10)
+    end
+
+    test "does not allow deposits of negative ammounts", %{bank: bank_pid}  do
```
Comment (nscyclone) [inline simple_bank/test/simple_bank_test.exs]: If I am not mistaken `ammount` should have a single `m`. This also occurs on lines 51 and 55.

Comment (doomspork) [reply simple_bank/test/simple_bank_test.exs]: You're not mistaken.  I just have awful spelling 😞

**Comment by doomspork** [comment]
I made some small changes to the code (like added guards to `deposit`).  Thoughts?  Is there anyway this could be improved to encourage folks to explore different part so the GenServer behavior?

File: simple_bank/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# SimpleBank
+
+In this lesson we're going to practice building a GenServer to
+
+- Capitalize all of the words in a given list
+
+  ```elixir
+  iex> Collections.capitalize(["elixir", "rocks"])
```
[the remark this answers is not in the document]
Comment (doomspork) [reply simple_bank/README.md]: It shouldn't be capitalize at all!  That's not part of the API.  This is copy pasta 😁

**Pull request merged**